libs/pmp-web/user/data-access/src/lib/services/user-data.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { map } from 'rxjs/operators';
+import { GetUserStatisticsCollectionPayload } from '../resources/payloads/get-user-statistics-collection.payload';
+import { ListUsersResponse, UserStatistics } from '@pimp-my-pr/shared/domain';
+import { urlFactory } from '@valueadd/typed-urls';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserDataService {
+  readonly endpoints = {
+    getUserStatisticsCollection: urlFactory('//localhost:3333/api/repository/users')
```

Comment:
We shouldn't hardcode API url in this way

Reply:
Yea, I know. Can it be done in another pull request to not make this one to complicated? Will it be enough when I add ToDo here?